core_backend/app/routers/admin.py

```diff
@@ -23,4 +28,32 @@ async def healthcheck(
             status_code=500, content={"message": f"Failed database connection: {e}"}
         )
 
+    if ALIGN_SCORE_METHOD == "AlignScore":
```

Comment:
Healthcheck always runs this (unless we change that env var) so even if we don't want alignscore and we spin up docker compose without the --profile alignScore tag, healthcheck will still fail unless we also set this var to None or smt. This behaviour should be changed or be added to docs clearly.

Reply:
In the next PR, I have provided an "LLM" option so we can just use that for AlignScore.